src/System/ref/Compat/System.cs

```diff
@@ -4995,14 +4995,13 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public CookieCollection() { }
         public int Count { get { return default(int); } }
         public bool IsReadOnly { get { return default(bool); } }
-        public bool IsSynchronized { get { return default(bool); } }
+        bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
```

Comment:
I'm not sure I understand why we needed to do this in this compat facade reference assembly. This seems to me like it might actually be a compat issue we need to fix in the implementation.